WcaOnRails/db/migrate/20200319193625_rename_combined_rounds_to_cutoff_rounds.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class RenameCombinedRoundsToCutoffRounds < ActiveRecord::Migration[5.2]
+  def change
+    # This change goes along renaming Combined Rounds to Cutoff Rounds in the seeds file to comply with 2020 WCA Regulations.
+    RoundTypes.delete_all
```

Comment:
`RoundTypes` -> `RoundType` =)

Reply:
Solved.
Now there must be a (big) mistake because running the migration deleted about half of the database file 😅